src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -403,7 +407,7 @@ public function getServiceIds()
         }
         $ids[] = 'service_container';
 
-        return array_unique(array_merge($ids, array_keys($this->methodMap), array_keys($this->services)));
+        return array_values(array_unique(array_merge($ids, array_keys($this->methodMap), array_keys($this->services))));
```

Comment:
i noted with https://github.com/symfony/symfony/pull/22801/files#diff-8aeba2b88d0347062017ee160a6beb49R141 keys are preserved; i want to do a clean assertion against `getServiceIds`, not relying on `array_values` or indexes in tests.

But can be left out yes; i just used this PR make the change as of 3.4 :)

Do you want a separate PR or rely on the current behavior?